evalml/tests/automl_tests/test_automl.py

```diff
@@ -3703,9 +3703,9 @@ def test_timeseries_baseline_init_with_correct_gap_max_delay(AutoMLTestEnv, ts_d
         problem_type="time series regression",
         problem_configuration={
             "date_index": "date",
-            "gap": 6,
+            "gap": 2,
             "max_delay": 3,
-            "forecast_horizon": 7,
+            "forecast_horizon": 1,
```

Comment:
Why did these values change?

Reply:
The dataset was too small so the splits are not compatible with the original values!

Reply:
Ahh, makes sense, thanks!